website/docs/extensions/starknet/contract-target.md

```diff
@@ -90,6 +90,35 @@ openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", br
 build-external-contracts = ["openzeppelin::account::account::Account"]
 ```
 
+## Wildcard support in build-external-contracts
+
+The `[build-external-contracts]` property explained above allows asking Scarb to build contracts from package dependencies as part of proper package build, for example for use in tests. This release adds a possibility to request building many contracts from a module tree at once. For example, the following snippet:
```

Comment:
```suggestion 
It is possible to request building many contracts from a module tree at once. For example, the following snippet:
```

1. I do not think there is a need to explain purpose of `build-external-contracts` here. 
1. Docs should not use expressions like "This release adds". It's not release notes.

Reply:
I agree with taking that part out.